packages/plugins/typescript-stencil-apollo/.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules
```

Comment:
Please add `example` here

Reply:
Oh sorry, I meant to add it to `.npmignore` because we don't want to publish it :) @ardatan

Reply:
Ok I see :) But I think you missed this one
https://github.com/dotansimha/graphql-code-generator/blob/stencil-apollo/packages/plugins/typescript-stencil-apollo/.npmignore